src/composables/PluginManager.js

```diff
@@ -0,0 +1,149 @@
+import { readTextFile } from 'src/composables/Files';
+import plugins from 'src/plugins';
+
+let instanciatePlugins = [];
+
+/**
+ * Initialize plugin metadata and definitions.
+ *
+ * @param {Plugin} plugin - instantiate plugin.
+ */
+export function initPluginMetadata(plugin) {
+  plugin.metadata = new plugin.pluginModel.PluginMetadata();
+
+  const definitions = plugin.metadata.getDefinitions();
+
+  // Add url file in svgImage for componentDefinitionCard
+  definitions.components.forEach((def) => {
+    if (def.icon) {
+      def.svgImage = `/plugins/${plugin.name}/icons/${def.icon}.svg`;
```

Comment:
not fan of this, because you override definition and may be you override something use by a specific plugin.
Is that possible to move this directly in ComponentDefinitionCard ?

Reply:
done